test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1956,6 +1959,28 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
             expect(test.form.el('factor-question').length).toEqual(1);
           });
       });
+      itp('calls transaction.authenticate with username, options and stateToken', function () {
+        return setupPasswordlessAuth(null, true, false).then(function (test) {
```

Comment:
what is the 3rd argument `false` for? I see only 2 arguments in `setupPasswordlessAuth` function

Reply:
Not needed, removed. It was the holding a flag for `idxStateToken`.